src/app/components/JsonSchemaConfigurator/JsonSchemaConfigurator.tsx

```diff
@@ -77,6 +78,15 @@ export const JsonSchemaConfigurator: FunctionComponent<JsonSchemaConfiguratorPro
           component: TypeaheadField,
         })
       : undefined;
+
+    forEach(otherProperties, (val: any, key: string) => {
```

Comment:
I think this and the existing code for the typeahead field is good for now.  If we start to have more custom components this will approach will require further development into some kind of very simple registry, also maybe there's ways we could get these pushed back to uniforms-patternfly.

Reply:
+1